tests/unit/converters/test_ome_zarr.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+import numpy as np
+import pytest
+import zarr
+
+from tiledbimg.converters.ome_zarr import OMEZarrReader
+
+
+@pytest.fixture
+def mocker_tiff(mocker):
+    mocker.patch("zarr.open")
+
+
+class TestOMEZarrReader:
+    def test_ome_zarr_level_count(self, tmp_path):
+        zarr_path = os.path.join(tmp_path, "test.zarr")
+        reader = OMEZarrReader(zarr_path)
+        reader._zarray = zarr.array([1, 2, 3, 4])
+        assert reader.level_count == 4
+        reader._zarray = zarr.array([1, 2, 3, 4, 5])
+        assert reader.level_count == 5
+
+    def test_ome_zarr_level_downsamples(self, tmp_path):
+        zarr_path = os.path.join(tmp_path, "test.zarr")
+        reader = OMEZarrReader(zarr_path)
+        assert reader.level_downsamples == ()
+
+    def test_ome_zarr_level_image(self, tmp_path, mocker_tiff):
+        zarr_path = os.path.join(tmp_path, "test.zarr")
+        reader = OMEZarrReader(zarr_path)
+        zarr_array = np.array(
+            [[[[[[[0, 1], [2, 3]], [[4, 5], [6, 7, 8]]]]]]], dtype=object
```

Comment:
```
In [5]: zarr_array
Out[5]: 
array([[[[[[list([0, 1]), list([2, 3])],
           [list([4, 5]), list([6, 7, 8])]]]]]], dtype=object)
```
That's not realistic data, the Ome-Zarr images are 5D arrays and do not hold nested lists:
```
In [18]: z = zarr.open("tests/data/CMU-1-Small-Region.ome.zarr/")

In [19]: z[0][0]
Out[19]: <zarr.core.Array '/0/0' (1, 3, 1, 2967, 2220) uint8>
```

I am sceptical of the utility of unit tests (at least for this repo) and this test illustrates why.

Reply:
IMO unit tests do not have to be realistic. On the realistic side at least to my mind are the integration (end-to-end) ones. Unit are there to ensure that the code actual output is the expected output according to input but no matter kind is the input. Of course since this repo pretty much wraps well known and tested functions like the ones from `numpy` etc... maybe it is unclear why we have them but in general testing parts of code even without real data is sth legit.

Reply:
They don't have to be realistic in the sense that they can be random-generated images but at least the array shape and dtype should be correct.

Also there's the issue of readability:
```
zarr_array = np.array(
    [[[[[[[0, 1], [2, 3]], [[4, 5], [6, 7, 8]]]]]]], dtype=object
)
...
expected = np.array(
    [[[[0, 1]], [[4, 5]]], [[[2, 3]], [[6, 7, 8]]]], dtype=object
)
```
I bet the `expected` value was produced by just running the test with the debugger and copying whatever result it returned, assuming it is correct. I strongly doubt a reviewer of test who just reads the input can come up with the expected result without running the code, it's not obvious at all.